src/helm/benchmark/window_services/test_palmyra_window_service.py

```diff
@@ -0,0 +1,161 @@
+import shutil
+import tempfile
+from typing import List
+
+from .tokenizer_service import TokenizerService
+from .window_service_factory import WindowServiceFactory
+from .test_utils import get_tokenizer_service, TEST_PROMPT
+
+
+class TestAntrhopicWindowService:
+    TEST_PROMPT_LENGTH: int = 51
+    TEST_TOKEN_IDS: List[int] = [
+        464,
+        3337,
+        329,
+        4992,
+        319,
+        5693,
+        32329,
+        357,
+        9419,
+        23264,
+        8,
+        318,
+        281,
+        987,
+        40625,
+        10219,
+        4642,
+        503,
+        286,
+        262,
+        13863,
+        5136,
+        329,
+        5524,
+        12,
+        19085,
+        1068,
+        35941,
+        9345,
+        357,
+        7801,
+        40,
+        8,
+        326,
+        12031,
+        284,
+        787,
+        7531,
+        14901,
+        287,
+        262,
+        2050,
+        11,
+        2478,
+        11,
+        290,
+        14833,
+        286,
+        8489,
+        4981,
+        13,
+    ]
+    TEST_TOKENS: List[str] = [
+        "The",
+        "ĠCenter",
```

Comment:
Argh. Do `response.tokens` from `PalmyraClient` also contain "Ġ"? Is this expected?

Reply:
Yes that is expected. That is what I got from calling directly their tokenizer